modules/common/react/lib/styles/accessibleHide.ts

```diff
@@ -9,7 +9,7 @@ const accessibleHide: CSSObject = {
   clipPath: 'polygon(0px 0px, 0px 0px, 0px 0px, 0px 0px)',
   position: 'absolute',
   overflow: 'hidden',
-  wordWrap: 'normal',
+  whiteSpace: 'nowrap',
```

Comment:
What was the reason for this change?

Reply:
linked a article at top from facebook engineer. 
TLDR: Some browser / screen reader combos don't read the space between words hidden with clip. Basically the words get newline separated and they don't read new lines as a space -> Hello World -> Hello{LineFeed}World
I read a few blogs today to make sure this is still the best method and this seems like the most bullet proof.